lib/styles.less

```diff
@@ -195,6 +195,12 @@ body {
         border-right: @style_chart_border_right;
     }
 
+    .sr-only {
+        position: absolute;
+        left:-9999px;
+        height: 1px;
+    }
```

Comment:
wonder if this might cause any issues? maybe in PDF export? is that really the recommended way to make something 'only for screenreaders'?

Reply:
yes, you can't set it to `display:none` and you can also not use `visibility:hidden`. we could also use an `aria-label` but that isn't always read out by screen readers and I don't understand the logic entirely.

see https://stackoverflow.com/a/26032207/1532965

Reply:
I think there might be another way but it would involve wrapping the `.dw-chart-body` div in another div in case the aria description is set:
```html
<div aria-label={ariaDescription}>
    <div id="chart" class="dw-chart-body" aria-hidden="true" />
</div>
```
but this would have two consequences:
* the user can't use HTML in the aria description which is unfortunate as things like tables or hyperlinks might provide useful context for a chart (specifically for visually impaired readers)
* it might mess with the rendering logic somehow, and we would need careful testing

Reply:
okay makes sense :+1: agree that keeping the door open for HTML there would be useful. would still be good to have a chart test covering this for PDF export